clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -14,6 +14,7 @@ import org.apache.spark.sql.functions._
 import org.apache.spark.sql.types.{StringType, StructField, StructType}
 import org.apache.spark.sql.{SparkSession, _}
 import com.amazonaws.services.s3.AmazonS3
+import io.treeverse.clients.StorageUtils._
```

Comment:
This is a bit scary, because it _looks_ like it could bring in an unlimited number of strange words.  In fact it isn't -- which also means it's not that useful.  I would _prefer_ (not required in any way, we have no style guide!) to remove this and use a `StorageUtils.` prefix wherever we need it.